includes/documents/abstract-wcpdf-order-document.php

```diff
@@ -418,12 +418,28 @@ public function get_title() {
 	}
 
 	public function get_number_title() {
-		$number_title = sprintf( __( '%s Number', 'woocommerce-pdf-invoices-packing-slips' ), $this->title );
+		if( $this->type == 'invoice' ) {
```

Comment:
This is the abstract, you should handle this in the classes themselves, and leave the function like you had it before (with `%`) as a fallback